### ISSUE TITLE
Can't add SSH remotes nor see existing ones

### ISSUE THREAD

**Issue by McSneaky** [microsoft/vscode-remote-release#8508]
Type: <b>Bug</b>

1. Launch VS Code
2. Click on "Add Remote" under Remote SSH options
3. Enter remote credentials
4. VS Code redirects back to Remote SSH plugin install page 

https://github.com/microsoft/vscode-remote-release/assets/6796697/71558d81-6f2d-498f-856a-b6f43a4c0784

Extension version: 0.102.0
VS Code version: Code 1.78.2 (b3e4e68a0bc097f0ae7907b217c1119af9e03435, 2023-05-10T14:39:26.248Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Sandboxed: Yes
Remote OS version: Linux x64 5.15.90.1-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 7950X 16-Core Processor             (32 x 4491)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|63.12GB (37.99GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+Ubuntu/home/mcsneaky/Projects/Cavai/Creative-Engine --remote=wsl+Ubuntu --crash-reporter-id 60b7470b-29de-49cb-b30b-83ebc0788826|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu|
|OS|Linux x64 5.15.90.1-microsoft-standard-WSL2|
|CPUs|AMD Ryzen 9 7950X 16-Core Processor (32 x 4491)|
|Memory (System)|30.91GB (27.01GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
282f8724:30602487
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
showlangstatbar:30737416
vsctsb:30705553
azdwalk:30721579
pythonms35:30701012
pythonfmttext:30731395
pythoncmv:30739700
fixhidewlkth:30730051
pythongtdpath:30739704

```

</details>

**Comment by ariliang** [comment]
same problem, vscode now cannot open ssh config file from wsl

**Comment by sudo-bussmann** [comment]
Also dropping in to say that I put off updating for a week or two. I upgraded and suddenly the process I had (pointing the `remote.SSH.path` to the helper .bat file that uses `C:\Windows\system32\wsl.exe %*` that allowed me to be able to use my SSH config from WSL no longer works. It was working fine until updating.

I've tried the various combos that should work and nothing seems to be fixing this issue.
